<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synonyms</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Suchen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputTa">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>279</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionSuchverlauf"/>
    <addaction name="separator"/>
    <addaction name="actionSchliessen"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSuchverlauf">
   <property name="text">
    <string>Suchverlauf</string>
   </property>
  </action>
  <action name="actionSchliessen">
   <property name="text">
    <string>Schliessen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSuchverlauf</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSchliessen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
